README.md

```diff
@@ -579,6 +579,7 @@ The following parameters will be the same for each process in the set:
 
 ### 2.8.0.0
 
+* Added Description and Parameter description for composite resources
```

Comment:
Please move this release note into the Unreleased section above. Version 2.8.0.0 has already been released.